runners/wikiConsole/templates/createDevon4jProject.asciidoc

```diff
@@ -0,0 +1,42 @@
+== Create a new Devon4j Project
+
+=== Create a new Devon4j Project from command prompt
+
+==== Prerequisites
+
+In order to create a new application you must use the archetype provided by devon4j which uses the https://maven.apache.org/guides/introduction/introduction-to-archetypes.html[maven archetype functionality].
+
+To create a new application, you should have installed devonfw IDE. Follow the https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-ide] documentation to install the same.
+
+==== Create the Project
+
+Now you can use devonfw to setup a java project for you by executing the following devon command in terminal.
+
+`devon java create <%= name; %>`
+
+==== What is generated
+
+The application template (archetype) generates a Maven multi-module project. It has the following modules:
```

Comment:
Sometimes you write Maven in lowercase letters and sometimes in uppercase letters. You should use the same writing everywhere. On the devonfw page the lower case version is used